buildPython.js

```diff
@@ -70,7 +70,9 @@ const saveFileName = (destFolder, fileName) => {
 		});
 
 		const activateCmd =
-			os.platform() === 'win32' ? 'venv\\Scripts\\activate.ps1' : '. venv/bin/activate';
+			os.platform() === 'win32'
```

Comment:
You can borrow some code from 
https://github.com/yaptide/ui/pull/1087/files#diff-8fde42b3d73fc28682a0aa0f818786554c7ae677ae6721de3ef8618bbfd42ef4

but still I am not sure if it works

Reply:
Can we withdraw that change, it can be possibly merge if https://github.com/yaptide/ui/pull/1087 is accepted by @ostatni5